certified-connectors/Calendly/apiDefinition.swagger.json

```diff
@@ -683,30 +1290,12 @@
       "properties": {
         "type": {
           "type": "string",
-          "enum": [
-            "ask_invitee",
-            "physical",
-            "outbound_call",
-            "inbound_call",
-            "custom",
-            "gotomeeting",
-            "zoom",
-            "google",
-            "microsoft_teams",
-            "webex"
-          ],
           "x-ms-summary": "Location type",
           "description": "Indicates the event location type.",
           "title": "Location Type"
         },
         "status": {
           "type": "string",
-          "enum": [
```

Comment:
The list of enum values has been removed from the input field. Existing flows may have this value chosen and will continue to send it.
can you please confirm that no users will be affected by this change?

Reply:
Yes, we can confirm that no users will be affected by this change.